File: dataset/human/java/file_2606.java
import java.util.*;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		String s = sc.next();
		String t = sc.next();
		
		char[] sd = s.toCharArray();
		char[] items = t.toCharArray();

		Character[] td = new Character[items.length];
		
		for(int i=0, len=items.length; i<len; i++) {
			td[i] = Character.valueOf(items[i]);
		}
		
		Arrays.sort(sd);
		Arrays.sort(td, Collections.reverseOrder());
		
		s = String.valueOf(sd);
		
		for(int i=0, len=items.length; i<len; i++) {
			items[i] = td[i].charValue();
		}
		
		t = String.valueOf(items);
		
		
		if(s.compareTo(t) <= -1) {
			System.out.println("Yes");
		}else {
			System.out.println("No");
		}
		
		sc.close();
	}
}

File: dataset/human/c/file_2606.c
#include<stdio.h>
int main(void)
{
char str[4];
scanf("%s",str);

int i,a,b,c;//a,b,cはそれぞれがあることを示す
a=b=c=0;
for(i=0;i<3;++i){//フラグ分岐
	if(str[i]=='a') a=1;
	if(str[i]=='b')	b=1;
	if(str[i]=='c')	c=1;
}

if(a==0||b==0||c==0) printf("No\n");
else printf("Yes\n");

return 0;
}
